src/engraving/style/styledef.h

```diff
@@ -215,6 +215,9 @@ enum class Sid {
     staffLineWidth,
     ledgerLineWidth,
     ledgerLineLength,
+    stemSlashHeight,
+    stemSlashAngle,
+    stemSlashWidth,

```

Comment:
When seeing `stemSlashHeight` and `stemSlashWidth` so close to each other, one might be confused about what these settings do. Have I understood correctly that `stemSlashHeight` determines the position, and `stemSlashWidth` the line thickness? Perhaps we could rename `stemSlashWidth` to `stemSlashThickness`, and `stemSlashHeight` to `stemSlashPosition`? (To me the word "height" seems to suggest that it has to do with the size of the slash itself, which is not the case.)
Let me know what you think.

Reply:
MuseScore historically has used 'width' rather than 'thickness' (mostly) in its settings/variables. But I agree that using 'thickness' would be much less confusing in anything new (and ideally, one day, when overhauling the old ones). SMuFL also uses 'thickness' in its `engravingDefaults`.

Reply:
Yup, I definitely prefer position and thickness, as you suggest, I just kept those name cause many other things are named that way, as Simon said. Gonna change it now.